docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -322,450 +405,821 @@ Storm supports a set of extended comparison operators (comparators) for speciali
+  *<min>* and less than *<max>*". This behavior differs from that of the ``*range=`` operator, which includes
+  **both** the minimum and maximum.
+- **Comparing intervals to intervals:** when using an interval with the ``@=`` operator to lift nodes based on
+  an interval property, Synapse returns nodes whose interval value has **any** overlap with the specified
+  interval. For example:
 
+  - A lift interval of September 1, 2018 to October 1, 2018 ( 2018/09/01, 2018/10/01 ) will match nodes with any
+    of the following intervals:
+  
+    - August 12, 2018 to September 6, 2018 ( 2018/08/12, 2018/09/06 ).
+    - September 13, 2018 to September 17, 2018 ( 2018/09/13, 2018/09/17 ).
+    - September 30, 2018 to November 5, 2018 ( 2018/09/30, 2018/11/05 ).
 
-*Lift using an interval with relative times:*
+- **Comparing intervals to times:** When using an interval with the ``@=`` operator to lift nodes based on a time
+  property, Synapse returns nodes whose time value falls within the specified interval.
+- **Comparing times to times:** When using a time with the ``@=`` operattor to lift nodes based on a time property,
+  Synapse returns nodes whose timestamp is an **exact match** of the specified time. While interval ( ``@=`` ) syntax is
+  supported for comparing times to other times, the regular equals comparator ( ``=`` ) can also be used and is more
+  straightforward.
+- When specifying date / time and interval values, Synapse allows the use of both lower resolution values (e.g., ``YYYY/MM/DD``),
+  and wildcard values (e.g., ``YYYY/MM*``). Wildcard time syntax may provide a simpler and more intuitive means to specify
+  some intervals. For example ``inet:whois:rec:asof=2018*`` is equivalent to ``inet:whois:rec:asof@=('2018/01/01', '2019/01/01')``.
+- Time-based keywords (such as ``now``) and relative time syntax (expressions such as ``+-1 hour`` or ``-7 days``) can be used
+  for interval values.
 
-- Lift the WHOIS email nodes that were observed between January 1, 2019 and the present:
+  See the type-specific documentation for :ref:`type-time` and :ref:`type-ival` types for a detailed discussion of these behaviors.
 
 
-.. storm-pre:: [inet:whois:email=(usnewssite.com, contact@privacyprotect.org) .seen=(2008/07/10 00:00:00.000, 2020/06/29 00:00:00.001) ]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: inet:whois:email.seen@=(2019/01/01, now)
+.. _lift-range:
 
+Lift by Range (\*range=)
+++++++++++++++++++++++++
 
-- Lift the DNS requests that occurred within one day after October 15, 2018:
+The range extended comparator (``*range=``) supports lifting nodes whose *<form>* = *<valu>* or *<prop>* = *<pval>* fall
+within a specified range of values. The comparator can be used with types such as integers and times.
 
+.. NOTE::
+  
+  The ``*range=`` operator can be used with both ``inet:ipv4`` and ``inet:ipv6`` values (which are stored
```

Comment:
IPv6 are not stored as integers but you can still use the `*range=` operator on them

Reply:
TIL